test/ffmpeg-vaapi/encode/encoder.py

```diff
@@ -202,3 +224,14 @@ def check_bitrate(self):
 
       # acceptable bitrate within 25% of minrate and 10% of maxrate
       assert(self.minrate * 0.75 <= bitrate_actual <= self.maxrate * 1.10)
+
+  def gen_r2r_md5(self):
```

Comment:
This is unnecessary.  Just get the md5 from the encoded file.  See example in other comment.

Reply:
removed it